components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -3000,10 +3002,14 @@ else if (APIConstants.API_KEY_TYPE_SANDBOX.equals(tokenType)) { // if
             if (StringUtils.isBlank(callbackUrl)) {
                 callbackUrl = null;
             }
+            String applicationTokenType = application.getTokenType();
+            if(StringUtils.isEmpty(application.getTokenType())){
+                applicationTokenType = APIConstants.APPLICATION_TOKEN_TYPE;
```

Comment:
Is this correct? What are the possible values for token type? Not only JWT and Default?